code/modules/worldgen/prefab/engines.dm

```diff
@@ -1,5 +1,17 @@
+#ifdef MAP_OVERRIDE_COGMAP1
```

Comment:
i am trading one hack for another here, but this should end up with only files relevant to the map being read/parsed.